ml-agents/mlagents/trainers/components/reward_signals/gail/signal.py

```diff
@@ -106,8 +146,9 @@ def prepare_update(
     ) -> Dict[tf.Tensor, Any]:
         """
         Prepare inputs for update. .
-        :param mini_batch_demo: A mini batch of expert trajectories
-        :param mini_batch_policy: A mini batch of trajectories sampled from the current policy
+        :param policy: The policy learning from GAIL signal
```

Comment:
Just randomly found this and fixed it. There are others that I can address separately.

Reply:
Wish there was a good way to automate checking this in CI, but don't think it's possible right now.